src/libraries/Common/src/Interop/Unix/System.Net.Security.Native/Interop.NetSecurityNative.cs

```diff
@@ -113,50 +112,50 @@ internal static partial class NetSecurityNative
             out uint retFlags,
             out bool isNtlmUsed);
 
-        [DllImport(Interop.Libraries.NetSecurityNative, EntryPoint="NetSecurityNative_DeleteSecContext")]
-        internal static unsafe extern Status DeleteSecContext(
+        [GeneratedDllImport(Interop.Libraries.NetSecurityNative, EntryPoint = "NetSecurityNative_DeleteSecContext")]
+        internal static unsafe partial Status DeleteSecContext(
             Status* minorStatus,
             IntPtr* contextHandle);
 
-        [GeneratedDllImport(Interop.Libraries.NetSecurityNative, EntryPoint="NetSecurityNative_GetUser")]
+        [GeneratedDllImport(Interop.Libraries.NetSecurityNative, EntryPoint = "NetSecurityNative_GetUser")]
         internal static partial Status GetUser(
-            out Status minorStatus,
+            Status* minorStatus,
             SafeGssContextHandle? acceptContextHandle,
             ref GssBuffer token);
 
-        [GeneratedDllImport(Interop.Libraries.NetSecurityNative, EntryPoint="NetSecurityNative_Wrap")]
+        [GeneratedDllImport(Interop.Libraries.NetSecurityNative, EntryPoint = "NetSecurityNative_Wrap")]
         private static unsafe partial Status Wrap(
-            out Status minorStatus,
+            Status* minorStatus,
             SafeGssContextHandle? contextHandle,
             bool isEncrypt,
             byte* inputBytes,
             int count,
             ref GssBuffer outBuffer);
 
-        [GeneratedDllImport(Interop.Libraries.NetSecurityNative, EntryPoint="NetSecurityNative_Unwrap")]
+        [GeneratedDllImport(Interop.Libraries.NetSecurityNative, EntryPoint = "NetSecurityNative_Unwrap")]
         private static partial Status Unwrap(
-            out Status minorStatus,
+            Status* minorStatus,
             SafeGssContextHandle? contextHandle,
             byte[] inputBytes,
             int offset,
             int count,
             ref GssBuffer outBuffer);
 
         internal static unsafe Status WrapBuffer(
-            out Status minorStatus,
+            Status* minorStatus,
```

Comment:
This isn't a native import so I'd expect it to have a natural managed API. So `out`, same for `UnwrapBuffer` below. It should be easy to add another `fixed` statement? Since this is internal I could also see the case for doing it this way.

Reply:
I switched these to be pointers as well because the other native import functions in this file are directly exposed as internal. I thought it was more in the spirit of consistency to make them all use `Status*` regardless of whether it is a native import or a wrapper around one..